src/makeRobot.js

```diff
@@ -38,7 +38,70 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const initialX = 0;
+  const initialY = 0;
```

Comment:
Looks clearer this way. Or also you can find a way to group them together
```suggestion
  const initialCoordinates = {
    x: 0,
    y: 0,
  };
  
  const evacuationCoordinates = {
    x: 1400,
    y: 500,
  };
```